tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/create_expected_ct_list_test.py

```diff
@@ -115,10 +116,34 @@ def setUp(self):
         """).render(project=self.project_id,
                     dataset=self.dataset_id,
                     table=OBSERVATION)
+
+        create_aian_table_tmpl = self.jinja_env.from_string("""
+        CREATE TABLE `{{project_id}}.{{dataset_id}}.{{table}}` AS (
+            SELECT person_id AS participant_id , research_id
+            FROM `{{project_id}}.{{dataset_id}}.{{map}}`
+        )
+        """).render(project_id=self.project_id,
+                    dataset_id=self.dataset_id,
+                    table=AIAN_LIST,
+                    map=PRIMARY_PID_RID_MAPPING)
```

Comment:
This should not be needed.  The table creation is handled by the base class when you set the list of table names via `cls.fq_table_names`.